sigstore/_verify.py

```diff
@@ -148,6 +151,18 @@ def verify(
             output(f"Failed to validate Rekor entry's SET: {inval_set}")
             continue
 
+        # 6) Verify that the signing certificate was valid at the time of signing
+        integrated_time = datetime.datetime.utcfromtimestamp(entry.integrated_time)
+        if (
+            integrated_time < cert.not_valid_before
+            or integrated_time > cert.not_valid_after
+        ):
+            # No need to log anything here.
+            #
+            # If an artifact has been signed multiple times, this will happen so it's not really an
+            # error case.
+            continue
+
```

Comment:
I might be misunderstanding the behavior here -- does this actually verify the timestamp? It seems as though it just `continue`s if the timestamp is invalid.

Reply:
IIUC, this is looking at every matching entry, and checking if any of the entries match the window. If none of them match, this loop will never get to the `valid_sig_exists = True` line and the validation will fail.